.github/workflows/.example.yml

```diff
@@ -0,0 +1,41 @@
+name: Example CI/CD
+
+on:
+  push:
+    branches: [ "master" ]
+    paths:
+      - 'example/**'
+  pull_request:
+    branches: [ "master" ]
+    paths:
+      - 'example/**'
+
+defaults:
+  run:
+    working-directory: ./example
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.x'
+          channel: 'stable'
+
+      - name: Install lcov
+        run: sudo apt-get install lcov
+
+      - name: Analyzing
+        run: flutter analyze
+
+      - name: Running test
+        run: flutter test --coverage
```

Comment:
do we have tests for example to check its coverage?

Reply:
No, we haven't. Have to I write them?